smoke-test/tests/cypress/integration_test.py

```diff
@@ -17,7 +17,7 @@ def ingest_cleanup_data():
 
 
 def test_run_cypress(frontend_session, wait_for_healthchecks):
-    command = f"npx cypress run --record"
+    command = f"npx cypress run"
```

Comment:
why is this change happening @eclaassen-pb ?

Reply:
@gabe-lyons that's how I was advised by @aditya-radhakrishnan to resolve a problem with Cypress failing to run.

```
Opening Cypress...
Your Record Key <redacted> is not valid with this projectId: <redacted>

It may have been recently revoked by you or another user.

Please log into the Dashboard to see the valid record keys.
```